app/assets/javascripts/components/reward_page_components/EditPersonLink.jsx

```diff
@@ -0,0 +1,33 @@
+class EditPersonLink extends React.Component {
+  constructor(props) {
+    super(props);
+    //this.timer = null;
+    this.timer = null;
+    this.add_emphasis = false;
+  }
+
+  decorate_link(url, text, person) {
+    if (!this.add_emphasis && person === undefined || person.iban !== null) {
+      return (<a href={url}>{text}</a>)
+    }
+
+    this.timer = setInterval(this.performTimerEvent.bind(this), 2000)
+    return (
+      <a href={url} className="tooltipped" data-position="botton" data-tooltip="Vul je bankrekeningnummer in om geld te verdienen!">
```

Comment:
Ze hebben het geld al verdiend. Misschien iets als "We hebben je bankrekeningnummer nodig om je beloning naar je te kunnen overmaken." Volgens mij is dat ook meer GDPR, want dan zeg je waar je de data voor gebruikt met een simpel voorbeeld.

Trouwens, is `botton` een geldige positie?